tools/twix/src/panels/enum_plot.rs

```diff
@@ -112,9 +112,14 @@ impl Panel for EnumPlotPanel {
 }
 
 impl EnumPlotPanel {
-    fn plot_segment(plot_ui: &mut PlotUi, segment: &Segment) {
+    fn plot_segment(plot_ui: &mut PlotUi, segment: &Segment, plot_bounds: &PlotBounds) {
         const VERTICAL_MARGIN: f64 = 0.05;
         const BORDER_WIDTH: f32 = 2.0;
+        const MIN_SEGMENT_RELATION_FOR_TEXT: f64 = 1.0 / 18.0;
+
+        let viewport_left_edge = plot_bounds.min()[0];
+        let viewport_right_edge = plot_bounds.max()[0];
+        let min_boxsize_text = viewport_right_edge - viewport_left_edge;
```

Comment:
Culling based on a fixed relative width is a bit brittle. Let's remove it for now and implement it properly after bumping the egui version.
```suggestion
        let viewport_left_edge = plot_bounds.min()[0];
```